.travis.yml

```diff
@@ -62,3 +62,8 @@ script:
   else
     echo "Skipped PHP_CodeSniffer due to insufficient PHP version: $(composer php:version)";
   fi;
+
+- >
+  echo;
+  echo "Running PHP-CS-Fixer";
+  composer ci:php:fixer;
```

Comment:
I wonder - as a separate issue/PR - if we might be able to have Travis just run `composer ci`, so that we can avoid the duplication.  The only issue I can see is requiring PHP 7 for CodeSniffer.  Maybe when we've dropped PHP 5.6 support...?

Reply:
This would make sense.

Currently, the `ci` script runs `ci:static` and `ci:dynamic`. We'd need to change to `ci` script to include the individual scripts, not `ci:static` and `ci:dynamic` anymore so the scripts can keep their current order of execution (which I'd like to do). Currently, the order of the scripts follows these principles:

1. heed the dependencies (e.g., if the PHP linting fails, executing or sniffing the PHP does not make sense anymore)
1. functional checks (e.g., unit tests) before stylistic issues
1. faster checks before slower checks (so the feedback is relatively fast)

Reply:
I've added #736 for this.